content/kubermatic/master/installation/start_kkp/_index.en.md

```diff
@@ -21,23 +21,13 @@ Follow the steps in [Guides]({{< ref "./guides/" >}}) for a detailed explanation
 Last but not least, there are a bunch of [Operations and Troubleshooting Guides]({{< ref "./cheat_sheets/" >}}) ready for you as well.
 
 ## Disclaimer
-This project is in early version at this stage.
+This project should serve mainly for the onboarding purposes and easiness to start with KKP.
 
-This project should serve mainly for the onboarding purposes and easiness to start with KKP. Make sure that you fully
-understand all parts and perform internal security checks before making your setup a production ready.
+Make sure that you fully understand all parts and perform internal security checks before making your setup a production ready.
 
-At this stage, there are following limitations:
- * only AWS and vSphere are available as the cloud providers
- * GitHub and GitLab are supported for storing and integration with GitOps tool
- * KKP CE edition is used.
-
-All the above limitations may change in upcoming iterations.
+At any stage or with more complex deployment options, feel free to [reach](https://www.kubermatic.com/contact-us/) the Kubermatic's Professional Services team.
```

Comment:
In addition to the PS team, I'd mention where folks can get open source support (e.g. Slack).

Reply:
done